dev/user.clj

```diff
@@ -0,0 +1,82 @@
+(ns dev.user
```

Comment:
It's not production code though, it is a test, but not one that we want to automate. We want to be able to run this bit of code to be able to see how the values change based on changes the ccm